src/main/java/org/jabref/logic/citationstyle/CSLAdapter.java

```diff
@@ -84,6 +85,14 @@ private void initialize(String newStyle, CitationStyleOutputFormat newFormat) th
          * Converts the {@link BibEntry} into {@link CSLItemData}.
          */
         private static CSLItemData bibEntryToCSLItemData(BibEntry bibEntry) {
+
+            // create a copy of bibEntry
+            bibEntry = (BibEntry) bibEntry.clone();
+            // change month field from "#mon#" to "mon"
+            if((bibEntry.getFieldMap().get(FieldName.MONTH) != null) && !bibEntry.getFieldMap().get(FieldName.MONTH).isEmpty()){
```

Comment:
Can you test if this works?

Reply:
Or add `LatexFieldFormatter fieldFormatter = new LatexFieldFormatter(...)` in front of the for loop and then add `.map(fieldFormatter ::format)`. In this way, we also support custom bibtex strings (and not only the predefined ones for the month field).

Reply:
Does not work, as LatexFieldFormatter a) needs preferences object b) has only a two argument format method and does not implment the interface

Reply:
ok, but it works "with obvious modifications" 😀

Reply:
@Siedlerchr I try ur suggestion but it didnt work.
I can also prefer your solution but instead of making this instruction inside if block 
```
bibEntry.getMonth().map(Month::getShortName).ifPresent(value -> bibTeXEntry.addField(new Key(key), new DigitStringValue(value)));
```
I will keep the instruction that I made.
```
bibEntry.getFieldMap().put(FieldName.MONTH, bibEntry.getMonth().get().getShortName());
```
What do you think?